changebot/blueprints/tests/test_stale_pull_requests.py

```diff
@@ -172,30 +180,29 @@ def test_close_disabled(self):
     def test_close_keep_open_label(self):
 
         # Time is beyond close deadline, and there is no comment yet but there
-        # is a keep-open label
+        # is a keep-open label so don't do anything.
 
         self.open_pull_requests.return_value = ['123']
         self.last_commit_date.return_value = now() - 241
+        self.last_comment_date.return_value = None
         self.find_comments.return_value = []
+        self.labels.return_value = ['keep-open']
 
         with app.app_context():
             with patch.object(app, 'stale_pull_requests_close', False):
                 # The list() call is to forge the generator to run fully
                 list(process_pull_requests('repo', 'installation'))
 
-        assert self.submit_comment.call_count == 1
-        expected = PULL_REQUESTS_CLOSE_WARNING.format(pasttime='4 minutes', futuretime='20 seconds')
-        self.submit_comment.assert_called_with(expected)
-        assert self.close.call_count == 0
-        assert self.set_labels.call_count == 0
+        assert self.submit_comment.call_count == 0
```

Comment:
Yes, I think this is now correct. Not sure why the test expected a warning with the label.